src/python/scripts/run_gerp.py

```diff
@@ -28,45 +28,81 @@
     $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw --gerp_exe_dir path/to/gerp/
 
 """
-
 import os
 import subprocess
+import argparse
+import json
 
-import argschema
 
+def add_ce_to_json(ce_file: str, json_file: str) -> None:
+    """Enrich the json file with the constrained elements
 
-class InputSchema(argschema.ArgSchema):
-    """Calculates the Genomic Evolutionary Rate Profiling (GERP) of a multiple sequence alignment (MSA)."""
+    Args:
+        ce_file: file containing the constrained elements.
+        json_file: json file with genome coordinate level.
 
-    msa_file = argschema.fields.InputFile(
-        required=True, description="MSA file (MFA format)"
-    )
-    tree_file = argschema.fields.InputFile(
-        required=True, description="Tree file (Newick format). Must include every species in the MSA."
-    )
-    depth_threshold = argschema.fields.Float(
-        required=False,
-        description="Constrained elements' depth threshold for shallow columns, in substitutions "
-                    "per site. By default, 0.5."
-    )
-    gerp_exe_dir = argschema.fields.InputDir(
-        required=False,
-        description="Path where 'gerpcol' and 'gerpelem' binaries can be found. By default, resort to $PATH."
-    )
+    Returns:
+        None
+    """
+    with open(json_file, 'r') as json_file_handler:
+        align_set = json.load(json_file_handler)
 
+    with open(ce_file) as ce_file_handler:
+        constrained_elements = [ce.split() for ce in ce_file_handler if ce.rstrip() != ""]
 
-if __name__ == "__main__":
-    mod = argschema.ArgSchemaParser(schema_type=InputSchema)
+    # enrich json file with constrained elements info
+    constrained_elems = []
+    for ce in constrained_elements:
+        constrained_elem = {}
+        constrained_elem["start"] = ce[0]
+        constrained_elem["end"] = ce[1]
+        constrained_elem["length"] = ce[2]
+        constrained_elem["score"] = ce[3]
+        constrained_elem["p-val"] = ce[4]
+        constrained_elems.append(constrained_elem)
+    align_set["constraint_elems"] = constrained_elems
+
+    with open(json_file, 'w') as json_file_handler:
+        json.dump(align_set, json_file_handler)
+
+
+def main(param: argparse.Namespace) -> None:
+    ''' Main function of the run_gerp.py script
```

Comment:
I'd suggest using a triple double-quoted string and ending with a period to be consistent with the docstring convention in PEP 257.
```suggestion
    """Main function of the run_gerp.py script.
```